build.gradle

```diff
@@ -43,15 +52,23 @@ subprojects {
     dependencyManagement {
         dependencies {
             dependency 'commons-logging:commons-logging:1.2'
+            dependency 'org.springframework.boot:spring-boot-starter:2.0.0.M3'
         }
     }
 
-    repositories {
-        jcenter()
+    dependencies {
+        testCompile('junit:junit')
+        testCompile('org.mockito:mockito-core')
+        testCompile('org.assertj:assertj-core')
+        testCompile('org.springframework.boot:spring-boot-starter-test')
+        testCompile('commons-logging:commons-logging')
```

Comment:
i think junit and mockito are transitive dependencies of spring-boot-starter-test, so you should be able to remove them

and why do we need commons-logging?